content/glossary/sql/having.md

```diff
@@ -0,0 +1,40 @@
+<!--
```

Comment:
remove "sql" from tags and subjects, title from markdown

Reply:
👍🏼